packages/dashboards/src/lib/configurator/components/formatters/raw-formatter/raw-formatter.component.ts

```diff
@@ -17,4 +17,12 @@ export class RawFormatterComponent {
     @Input()
     @HostBinding("class")
     public elementClass: string;
+
+    public get displayValue(): string {
+        const valueAsNumber = parseFloat(this.data?.value);
+        if (isNaN(valueAsNumber)) {
+            return this.data?.value;
+        }
+        return valueAsNumber.toLocaleString();
```

Comment:
`ngOnChanges` preferred for this. `toLocaleString` might be quite expensive and parseFloat is the same. Overall there's quite a lot going on here.